tensorboard/webapp/runs/views/runs_table/runs_data_table.scss

```diff
@@ -48,11 +48,13 @@ $_arrow_size: 16px;
 
 tb-data-table-content-row,
 tb-data-table-header-cell {
-  height: 44px;
+  // Match the size of the touch targets for the checkbox.
+  height: 48px;
```

Comment:
I wish the touch target would match our size like all the other parts of mdc-checkbox do. I am ok with this though.